replicate/rolebinding/rolebindings.go

```diff
@@ -159,6 +166,19 @@ func (r *Replicator) ReplicateObjectTo(sourceObj interface{}, target *v1.Namespa
 	return nil
 }
 
+//Checks if Role required for RoleBinding exits. Retries a few times before returning error to allow replication to catch up
+func (r *Replicator) canReplicate(targetNameSpace string, roleRef string) (err error) {
+	for i := 0; i < 5; i++ {
```

Comment:
I'm not all too happy with this change, because it's basically just brute-forcing the object creation until it accidentally succeeds. However, I recognize that any more sophisticated solution would probably require quite a bit more refactoring on the existing code base... 😩